Library/Tracking/KSRAnalyticsTests.swift

```diff
@@ -414,6 +414,24 @@ final class KSRAnalyticsTests: TestCase {
     XCTAssertEqual("sign_up", segmentClient.properties.last?["context_page"] as? String)
   }
 
+  func testTrackLoginPageViewed() {
+    let dataLakeClient = MockTrackingClient()
+    let segmentClient = MockTrackingClient()
+    let ksrAnalytics = KSRAnalytics(
+      dataLakeClient: dataLakeClient,
+      loggedInUser: nil,
+      segmentClient: segmentClient
+    )
+
+    ksrAnalytics.trackLoginPageViewed()
```

Comment:
Can you add this test into the function defined on line 2442 of this file? It looks like we add/test all the events into there.